### PR TITLE
Create boolean admin column in users

### PR DESCRIPTION
**Pull request by dvanetten03**
Created boolen admin column in users.
Set up factory_girl :user

### CODE REVIEW

File: db/migrate/20160331232622_devise_create_users.rb [hunk 1]
```diff
@@ -1,42 +0,0 @@
-class DeviseCreateUsers < ActiveRecord::Migration
```
Comment (kenmazaika) [inline db/migrate/20160331232622_devise_create_users.rb]: @dvanetten03 how come this file needs to be deleted? Do we have two of the same migration file in the app?

**Comment by kenmazaika** [comment]
Looks good, but I had one question prior to merging this

File: db/migrate/20160331232622_devise_create_users.rb (hunk 1 above)
Comment (dvanetten03) [inline db/migrate/20160331232622_devise_create_users.rb]: No, unfortunately, I did not have the correct 'master' when I first merged into my account and I had already 'migrated' Devise...my mistake.

Comment (kenmazaika) [inline db/migrate/20160331232622_devise_create_users.rb]: Then I think we want to continue to have this file in (we should re-add it to the branch).  To get the migrations to work on your system it probably makes sense to rebuild the database with the migrations only on the branch.  Here's how you can do that.

First, stop your server and any rails consoles. 

Second, run this command (and make sure you don't see any error messages when you run it):

```
rake db:drop
```
 
Third, run this command

```
rake db:create
```

Fourth, run this command:

```
rake db:migrate
```

**Comment by dvanetten03** [comment]
This is ready now as ...devise_create_users.rb has been added back to repo.

**Comment by kenmazaika** [comment]
Looks great! :+1:

**Pull request merged**